### PR TITLE
add ticket 6 with error halnding and endpoint description

### PR DESCRIPTION
**Pull request by crepsleyfox**

### CODE REVIEW

File: 1-controllers/comments-controller.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { selectCommentsByArticleId } = require("../1-models/comments-model")
+
+
+exports.getCommentByArticleId = (req, res, next) => {
+    const { article_id } = req.params
+
+    selectCommentsByArticleId(article_id)
+    .then((comments) => {
+        console.log(comments)
```
Comment (nc-verity-gregory) [inline 1-controllers/comments-controller.js:9]: Don't leave console logs in your code!!

File: 1-models/comments-model.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+exports.selectCommentsByArticleId = (article_id) => {
+    const queryString = `
+        SELECT comment_id, votes, created_at, author, body, article_id
+        FROM comments
+        WHERE article_id = $1
+        ORDER BY created_at DESC;`
+
+    return db.query(queryString, [article_id]).then(({ rows }) => {
+        if (rows.length === 0) {
```
Comment (nc-verity-gregory) [inline 1-models/comments-model.js:11]: Is it an error if no comments are returned? What about new articles that won't have comments yet?

File: __tests__/app.test.js [hunk 3]
```diff
@@ -113,4 +127,31 @@ describe("NC News Server", () => {
...
+    })
+    test('GET 200 : return all comments for a specific article_id in descending order by date comment was posted', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.comments.length).toBe(11)
+            expect(body.comments).toBeSortedBy('created_at', { descending: true })
+        })
+    })
```
Comment (nc-verity-gregory) [inline __tests__/app.test.js:155]: What about an article that has no comments? What would that status code be? (Article 2 has no comments)

**Review by nc-verity-gregory — CHANGES_REQUESTED** [review]
Really good start, just that one test missing for a valid article with no ID :) Please add it then resubmit.

**Review by poppymcd — CHANGES_REQUESTED** [review]
Great job, merge when ready!

**Review by poppymcd — APPROVED** [review]
Great job, merge when ready!

**Pull request merged**